src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <>
```

Comment:
redundant fragment - should be removed
fix _all_ similar cases
![image](https://github.com/mate-academy/react_static-list-of-todos-js/assets/86923089/4ec8e768-8637-439f-a97f-ae2f5a67a446)